Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:bionic
```

Comment:
https://github.com/parsable/docker/blob/main/python/python3-builder.Dockerfile
I wonder if we can just base this off our internal python3 image. But I think Luba would be better fit to direct you on that.